bundles/org.openhab.binding.mqtt.homeassistant/src/main/java/org/openhab/binding/mqtt/homeassistant/internal/component/Climate.java

```diff
@@ -44,20 +44,25 @@
  *
  * @author David Graeff - Initial contribution
  * @author Anton Kharuzhy - Implementation
+ * @author Vaclav Cejka - added support for humidity and preset_modes
  */
 @NonNullByDefault
 public class Climate extends AbstractComponent<Climate.ChannelConfiguration> {
     public static final String ACTION_CH_ID = "action";
     public static final String AUX_CH_ID = "aux";
     public static final String AWAY_MODE_CH_ID = "away-mode";
     public static final String AWAY_MODE_CH_ID_DEPRECATED = "awayMode";
+    public static final String CURRENT_HUMIDITY_CH_ID = "current-humidity";
+    public static final String CURRENT_HUMIDITY_CH_ID_DEPRECATED = "currentHumidity";
```

Comment:
since there has never been a currentHumidity channel, we don't need to maintain it as deprecated. just remove this line (and change the conditional below that uses it to only use `CURRENT_HUMIDITY_CH_ID`)

Reply:
thanks for all points, good school for me...
Done